internal/permissions/model.conf

```diff
@@ -18,8 +18,7 @@ e = some(where (p.eft == allow)) \
 
 [matchers]
 # req subject must inherit from policy subject
+# req owner must match policy owner
```

Comment:
i'm a little confused ... what's the difference between subject and owner?

Reply:
subject is the person or group the policy is affecting. Owner is the person that owns the record. For example in "Alice has permission for Bob's document", Alice is subject and Bob is owner